common/app/views/fragments/atoms/storyquestions.scala.html

```diff
@@ -26,7 +27,71 @@
     }
 }
 
-@if(!isAmp && !isClosed) {
+@if(!isAmp && !isClosed && ReaderAnswersDeliveryMechanism.isSwitchedOn) {
+    <div id="user__question-atom-@storyquestions.id" class="js-view-tracking-component submeta user__question">
+
+        <span class="js-storyquestion-atom-id is-hidden" id="@storyquestions.id"></span>
+        <span class="is-hidden" id="js-storyquestion-is-answer-delivery-test-ready" data-is-answer-delivery-test-ready="@ReaderAnswersDeliveryMechanism.isSwitchedOn"></span>
```

Comment:
This is an FYI because the error has been made before this PR, but for metadata like this it is usually much easier to just add data attributes to the global container, i.e.
```html
<div id="user__question-atom-@storyquestions.id"
    data-atom-id="@storyquestions.id"
    data-is-answer-delivery-test-ready="@ReaderAnswersDeliveryMechanism.isSwitchedOn"
    >
```